src/main/java/tc/oc/pgm/listeners/ServerPingDataListener.java

```diff
@@ -101,40 +101,34 @@ private void serializeMatch(Match match, JsonObject jsonObject) {
   private void appendNextMap(JsonObject jsonObject) {
     checkNotNull(jsonObject);
 
-    PGMMapOrder mapOrder = this.matchManager.getMapOrder();
-    if (mapOrder != null) {
-      PGMMap nextMap = mapOrder.getNextMap();
+    MapInfo nextMap = mapOrder.getNextMap();
 
-      if (nextMap != null) {
-        JsonObject nextMapObject = new JsonObject();
-        this.serializeMap(nextMap, nextMapObject);
-        jsonObject.add("next_map", nextMapObject);
-      }
+    if (nextMap != null) {
+      JsonObject nextMapObject = new JsonObject();
+      this.serializeMap(nextMap, nextMapObject);
+      jsonObject.add("next_map", nextMapObject);
     }
   }
 
-  private void serializeMap(PGMMap map, JsonObject jsonObject) {
+  private void serializeMap(MapInfo map, JsonObject jsonObject) {
     checkNotNull(map);
     checkNotNull(jsonObject);
 
-    MapInfo mapInfo = map.getInfo();
-    MapPersistentContext persistentContext = map.getPersistentContext();
-
-    jsonObject.addProperty("slug", mapInfo.slug());
-    jsonObject.addProperty("name", mapInfo.name);
-    jsonObject.addProperty("version", mapInfo.version.toString());
-    jsonObject.addProperty("objective", mapInfo.objective);
+    jsonObject.addProperty("id", map.getId());
```

Comment:
This is going to break at least OCC website. The slug is used to locate map icons afaik. Can we leave `slug` here, please?

Reply:
`<slug>` is becoming `<id>` so I think we'll have to accept this breaking change.